docs/en/ingest-management/release-notes/release-notes-8.13.asciidoc

```diff
@@ -22,6 +23,24 @@ Also see:
 * {kibana-ref}/release-notes.html[{kib} release notes]
 * {beats-ref}/release-notes.html[{beats} release notes]
 
+// begin 8.13.2 relnotes
+
+[[release-notes-8.13.2]]
+== {fleet} and {agent} 8.13.2
+
+Review important information about {fleet-server} and {agent} for the 8.13.2 release.
+
+[discrete]
+[[bug-fixes-8.13.2]]
+=== Bug fixes
+
+{fleet}::
+* Fixes having to wait ten minutes after agent upgrade if agent cleared watching state ({kibana-pull}179917[#179917]).
+* Fixes using the latest compatible version in K8's manifest instead of latest available version ({kibana-pull}179662[#179662]).
```

Comment:
This should actually be inverted, e.g. 

```
Fixes using the latest available version in K8's manifest instead of latest compatible version
```

The PR linked makes it such that non-compatible versions of Agent (e.g. newer minor release) won't appear in the Fleet K8's manifest anymore.

Reply:
Thanks @kpollich. This is fixed in [c877f22](https://github.com/elastic/ingest-docs/pull/1003/commits/c877f22260078be04843c5fe2fae643f810413f3)

I'll add a comment to the original Kibana PR so that this gets fixed there too.